src/workerd/api/blob.h

```diff
@@ -55,8 +56,13 @@ class Blob: public jsg::Object {
 
     JSG_METHOD(slice);
     JSG_METHOD(arrayBuffer);
+    JSG_METHOD(bytes);
     JSG_METHOD(text);
     JSG_METHOD(stream);
+
+    JSG_TS_OVERRIDE({
+      bytes(): Uint8Array;
```

Comment:
It's way more likely you're right than I am, but why shouldn't this be a `Promise<Uint8Array>`?

Same question for the TS override in http.h as well

Reply:
Doh. Yes lol